### DISCUSSION TITLE
DataEditR support for 4.2+

### DISCUSSION THREAD

**Discussion by macsmith26** [DillonHammill/DataEditR#55] [Ideas]
Hi, 

Any chance this package will be updated to support 4.2+ in the near future?

**Comment by DillonHammill** [comment]
@macsmith26 are you referring to R version 4.2.0? DataEditR was built with R 4.0.0 and so it should work fine, have you encountered any issues for version 4.2.0 of R?

**Reply by macsmith26** [reply]
CRAN install didn't work for me prior to comment, but it's working now. All good, thanks!
